source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,107 @@
+   As long as the existing deployment is not to the root of a drive, a new folder on the same drive can work for the new deployment.
+   If the existing standalone system points to the root of the drive, you must use a separate drive for the new deployment.
+
+   Use the environment variable values retrieved from the standalone deployment to establish the same values for the new deployment.
+
+   Set the port to a custom point different than the existing standalone deployment.
+
+#. Duplicate **configurations** from existing standalone deployment to new deployment
+
+
+
+#. Duplicate **buckets** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc ls` with the ``--json`` flag to retrieve a list of the buckets that exist on the standalone deployment.
+   Use the list to recreate the buckets on the new deployment.
+
+#. Duplicate **tiers** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin tier ls` with the ``--json`` flag to retrieve a list of the tiers that exist on the standalone deployment.
+   Use the list to recreate the tiers on the new deployment.
+
+#. Duplicate **policies** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin policy list` with the ``--json`` flag  to retrieve a list of policies that exist on the standalone deployment.
+   Use the list to recreate the policies on the new deployment.
+
+#. Duplicate **groups** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin group list` with the ``--json`` flag to retrieve a list of groups that exist on the standalone deployment.
+   Use the list to recreate the groups on the new deployment.
+
+#. Duplicate **users** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user list` with the ``--json`` flag to retrieve a list of users with access key, policy name, and status.
+   Use the list to recreate the users on the new deployment.
+
+   Note: You will need to define each user's ``SECRETKEY`` on the new deployment.
+   Make note of the ``SECRETKEY`` for each user to let them know their new credentials.
+
+#. Duplicate **service accounts** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin user svcacct ls` with the ``--json`` flag to list existing service accounts on the standalone deployment.
+   Use the list to recreate the service accounts on the new deployment.
+
+#. Use :mc:`mc mirror` with the ``--watch`` flag on the standalone deployment to move objects to the new |SNSD| deployment
+
+   Do this for each bucket.
+
+#. After the initial mirror process completes, convert standalone deployment to be read only
+
+   Some options for doing this include:
+
+   - ``mc admin policy remove`` for each active policy
+   - ``mc admin policy unset`` to remove policy/policies for each group
+
+#. Wait for ``mc mirror`` to complete for all buckets.
```

Comment:
I think this will fail if you remove policy as above correct? (For all subsequent requests)